src/includes/performance/default-sampling-context/python.aiohttp.mdx

```diff
@@ -0,0 +1,9 @@
+<PlatformContent includePath="performance/default-sampling-context-platform" />
+
+The AIOHTTP integration adds the incoming request object:
+
+```python
+{
+  "aiohttp_request": aiohttp.web_request.Request
```

Comment:
Can we change this to `<aiohttp.web_request.Request>` or something that indicates we are passing an instance. Likewise with the others.

Reply:
Sure. (I'll admit it's more obvious with the primitives.)

Reply:
Yeah, I get that this is probably convention in Typescript but in Python this notation hasn't really caught on yet.